### PR TITLE
StudentExamFees code update

### PR DESCRIPTION
**Pull request by viralpandya**

### CODE REVIEW

File: OtripleS.Web.Api/Controllers/StudentExamFeesController.cs [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+        //    }
+        //    catch (StudentExamFeeDependencyException studentExamFeeDependencyException)
+        //    {
+        //        return Problem(studentExamFeeDependencyException.Message);
+        //    }
+        //    catch (StudentExamFeeServiceException studentExamFeeServiceException)
+        //    {
+        //        return Problem(studentExamFeeServiceException.Message);
+        //    }
+        //}
```
Comment (hassanhabib) [inline OtripleS.Web.Api/Controllers/StudentExamFeesController.cs:110]: @viralpandya you can't have that in here.

Comment (viralpandya) [reply OtripleS.Web.Api/Controllers/StudentExamFeesController.cs]: Resolved

File: OtripleS.Web.Api/Services/StudentExamFees/StudentExamFeeService.Validations.cs [hunk 2]
```diff
@@ -12,30 +12,30 @@ namespace OtripleS.Web.Api.Services.StudentExamFees
...
-        }
+        //private void ValidateStudentExamFeeId(Guid studentExamFeeId)
+        //{
+        //    if (studentExamFeeId == default)
+        //    {
+        //        throw new InvalidStudentExamFeeException(
+        //            parameterName: nameof(StudentExamFee.Id),
+        //            parameterValue: studentExamFeeId);
+        //    }
+        //}
```
Comment (hassanhabib) [inline OtripleS.Web.Api/Services/StudentExamFees/StudentExamFeeService.Validations.cs:23]: @viralpandya go through your code and make sure you don't have any commented code in your PRs please.

Comment (viralpandya) [reply OtripleS.Web.Api/Services/StudentExamFees/StudentExamFeeService.Validations.cs:23]: fixed this one but not in this PR. Removing it at 4 pm

Comment (viralpandya) [reply OtripleS.Web.Api/Services/StudentExamFees/StudentExamFeeService.Validations.cs]: removed

**Pull request merged**